rosidl_generator_py/CMakeLists.txt

```diff
@@ -46,10 +47,12 @@ if(AMENT_ENABLE_TESTING)
 
   rosidl_generate_interfaces(${PROJECT_NAME} ${message_files}
     SKIP_INSTALL
+    TRACE
   )
 
   ament_add_nose_test(test_interfaces_py "test/test_interfaces.py"
-    APPEND_ENV PYTHONPATH=${CMAKE_CURRENT_BINARY_DIR}/rosidl_generator_py:${CMAKE_CURRENT_SOURCE_DIR}
+    APPEND_ENV PYTHONPATH="${CMAKE_CURRENT_BINARY_DIR}/rosidl_generator_py:${CMAKE_CURRENT_SOURCE_DIR}"
+    APPEND_LIBRARY_DIRS "${CMAKE_CURRENT_BINARY_DIR}/Release"
```

Comment:
What is this library dir needed for? If it is only necessary for Windows it should also only be added for that platform? And if it points to the folder on Windows which contains the compiled target this needs to handle the currently selected configuration rather then hardcoded `Release`.